Automattic/wp-calypso
# Add/new email comparison card props for google workspace

#### Changes proposed in this Pull Request

Initial code for the Google Workspace Card, it is just a visual component, work for adding item to the cart will be done on a subsequent pull request. I'm just breaking down things for smaller reviews and testings.

#### Testing instructions

1. Go to a site with domains without email subscription
2. Click on 'Add email' for the desired site
3. Once you are in the comparison page, add at the end the following flag: `?flags=emails/new-email-comparison`

You should see the beginning of a new component being developed:
![image](https://user-images.githubusercontent.com/5689927/144403739-b3bcf7e3-9bf1-4552-81f5-7a35283f2462.png)

Adding to cart won't work, it will be added in a subsequent pull request

Related to {1200182182542585-as-1201439368870020}
